src/feature/stats/rephist.c

```diff
@@ -1527,6 +1533,52 @@ rep_hist_load_state(or_state_t *state, char **err)
   return 0;
 }
 
+/** Collects BandwidthStatistics parameter from consensus.
+ *  Returns 0 iff BandwidthStatistics is 0 or
+ *  the consensus param is 0. */
+bool
+get_bandwidth_stats_param(void)
```

Comment:
There's a bunch of confusing things happening in this function.

I think it would be simpler, and easier to read, if you rewrote it like the example function:
https://github.com/torproject/tor/blob/44f92e8e4278403b9e633668f8be70f197b6e8db/src/core/or/dos.c#L154

Reply:
We use a similar structure for our function comments, so people can understand them easily.
Please copy the comment from the example function.
Then make small changes to describe what this function actually does.